pkg/loop/config.go

```diff
@@ -70,6 +70,9 @@ const (
 	envTelemetryLogExportMaxBatchSize     = "CL_TELEMETRY_LOG_EXPORT_MAX_BATCH_SIZE"
 	envTelemetryLogExportInterval         = "CL_TELEMETRY_LOG_EXPORT_INTERVAL"
 	envTelemetryLogMaxQueueSize           = "CL_TELEMETRY_LOG_MAX_QUEUE_SIZE"
+	envTelemetryTraceCompressionDisabled  = "CL_TELEMETRY_TRACE_COMPRESSION_DISABLED"
+	envTelemetryMetricCompressionDisabled = "CL_TELEMETRY_METRIC_COMPRESSION_DISABLED"
+	envTelemetryLogCompressionDisabled    = "CL_TELEMETRY_LOG_COMPRESSION_DISABLED"
```

Comment:
We have to avoided negative field names like `*Disabled` to prevent confusion from double-negatives. Can we find a different way to do this? If we are just mapping to a string to pass along, how about just exposing that to stay flexible?
```suggestion
	envTelemetryTraceCompression  = "CL_TELEMETRY_TRACE_COMPRESSION"
	envTelemetryMetricCompression = "CL_TELEMETRY_METRIC_COMPRESSION"
	envTelemetryLogCompression    = "CL_TELEMETRY_LOG_COMPRESSION"
```
I think it still OK to default to `gzip` if that is important (though it may not be since they are always launched by a host), but I suppose in that case we should have an explicit `none` or similar.